library/date/friday_spec.rb

```diff
@@ -0,0 +1,8 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+require 'date'
+
+describe "Date#friday?" do
+  it "should be friday" do
+    Date.new(2000, 1, 7).friday?.should be_true
+  end
```

Comment:
To test this better, it would be interesting to check that `friday?` returns false for dates which are not on friday.